### PR TITLE
Modernize ArtistInspector a bit...

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/artist.py [hunk 1]
```diff
@@ -11,22 +11,6 @@
 from .path import Path
 from .transforms import (Bbox, IdentityTransform, Transform, TransformedBbox,
                          TransformedPatchPath, TransformedPath)
-# Note, matplotlib artists use the doc strings for set and get
```
Comment (anntzer) [inline lib/matplotlib/artist.py]: that comment is essentially completely false now (even the magic string for aliases is "Alias for", not "alias for" :))

File: lib/matplotlib/artist.py [hunk 2]
```diff
@@ -1296,34 +1274,25 @@ def get_setters(self):
...
         property, which does not, return 'transform'
         """
-
-        if s in self.aliasd:
-            return s + ''.join([' or %s' % x
-                                for x in sorted(self.aliasd[s])])
-        else:
-            return s
+        return s + ''.join(' or %s' % x
+                           for x in sorted(self.aliasd.get(s, [])))
```
Comment (timhoffm) [inline lib/matplotlib/artist.py]: Just to keep this structurally similar to `alias_name_rest()`:
~~~
aliases = ''.join(' or %s' % x for x in sorted(self.aliasd.get(s, [])))
return s + aliases
~~~
Also slightly more readable.

File: lib/matplotlib/artist.py [hunk 3]
```diff
@@ -1296,34 +1274,25 @@ def get_setters(self):
...
         ds = o.__doc__
         if ds is None:
             return False
         return ds.startswith('Alias for ')
 
     def aliased_name(self, s):
         """
-        return 'PROPNAME or alias' if *s* has an alias, else return
-        PROPNAME.
+        return 'PROPNAME or alias' if *s* has an alias, else return PROPNAME.
```
Comment (timhoffm) [inline lib/matplotlib/artist.py]: May give it a capital to start, when you're editing anyway. :smile:

**Comment by anntzer** [comment]
comments handled

**Comment by NelleV** [comment]
Thanks @anntzer

**Pull request merged**